wp-content/themes/theme_name/skin/assets/images/logo.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="186" height="26" viewBox="0 0 186 26">
```

Comment:
No, because there is no need for that. This logo is here just as a place holder so when you set things up, you don't get no image placeholder. No one is going to use Infinum Logo at his project. And this is going to be the first thing that you will delete.

So no because there is no need for that :D